src/helpers/validate.js

```diff
@@ -9,7 +9,7 @@ function validateURL(url) {
 }
 
 function validateFlags(flags) {
-  const mask = /^(?!-)[\w,-]+$/
+  const mask = /^[\w,\.,\-]+{1,45}$/
```

Comment:
There is benefit in moving this regex outside.

Currently, every invocation of `validateFlags` will instantiate a unique regular expression instance which will promptly be destroyed (and never reused) after the function returns. The optimization here is to simply instantiate one regular expression but share it among every invocation of the function.